zio/parquetio/reader.go

```diff
@@ -360,18 +363,28 @@ func convertListType(els []*parquet.SchemaElement, i int) (int, *parquetColumn,
 }
 
 func (r *Reader) Read() (*zng.Record, error) {
-	if r.record == int(r.pr.GetNumRows()) {
-		return nil, nil
-	}
-	r.record++
+	var err error
```

Comment:
Nit: Move down so it's adjacent to `r.buf, err = r.pr.ReadByNumber(toread)`.

Reply:
this also gets used down on line 403, we can create another local there?  i assume they compile to the same thing, neither one seems dramatically better to me, do you prefer the two separate instances?

Reply:
Doh! Missed the other use. The generated code will almost certainly be identical. I like the two locals approach, but what's here is also totally fine.